.github/workflows/smoke-test.yml

```
@@ -0,0 +1,68 @@
+name: Smoke Test Docker Image
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+env:
+  ORG: timescaledev
+  TS_VERSION: main
+  PLATFORM: linux/amd64
+
+jobs:
+  smoketest:
+    name: PG${{ matrix.pg }}-${{ matrix.type }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [14]
```

i think we wanna test all the versions we build here 
```suggestion
        pg: [12,13,14,15]
```